recipes/valideer/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.4.2" %}
+
+package:
+    name: valideer
+    version: {{ version }}
+
+source:
+    fn: valideer-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/v/valideer/valideer-{{ version }}.tar.gz
+    md5: 9129e0c2394e2b30947f07c5e1bf5d4a
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - decorator
+
+test:
+    imports:
+        - valideer
+        - valideer.tests
+
+about:
+    home: https://github.com/podio/valideer
+    license: MIT
+    summary: Lightweight data validation and adaptation library for Python
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
+        - emiliom
```

Comment:
@emiliom when I add your GitHub handle here as a maintainer you'll be automatically added to this recipe team. Let me know if that is OK.

(I am adding you here because this recipe is of interest to ODM2.)